src/askui/models/router.py

```diff
@@ -73,46 +73,46 @@ def __init__(self, log_level, report,
         self.huggingface_spaces = HFSpacesHandler()
         self.tars = UITarsAPIHandler(self.report)
     
-    def act(self, controller_client, instruction: str, model_name: str | None = None):
+    def act(self, controller_client, locator: str, model_name: str | None = None):
         if self.tars.authenticated and model_name == "tars":
-            return self.tars.act(controller_client, instruction)
+            return self.tars.act(controller_client, locator)
         if self.claude.authenticated and model_name == "claude":
             agent = ClaudeComputerAgent(controller_client, self.report)
-            agent.run(instruction)
+            agent.run(locator)
         raise AutomationError("Invalid model name for act")
     
-    def get_inference(self, screenshot: Image.Image, instruction: str, model_name: str | None = None):
+    def get_inference(self, screenshot: Image.Image, locator: str, model_name: str | None = None):
         if self.tars.authenticated and model_name == "tars":
-            return self.tars.get_prediction(screenshot, instruction)
+            return self.tars.get_prediction(screenshot, locator)
         if self.claude.authenticated and model_name == "anthropic-claude-3-5-sonnet-20241022":
-            return self.claude.get_inference(screenshot, instruction)
+            return self.claude.get_inference(screenshot, locator)
         raise AutomationError("Executing get commands requires to authenticate with an Automation Model Provider supporting it.")
```

Comment:
I think for the `get_inference` it is actually a `query` (or similar) and not a `locator`